ui/app/AppLayouts/Profile/panels/CommunityPanel.qml

```diff
@@ -0,0 +1,84 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+
+Item {
+    id: root
+    property alias title: statusCommunityItem.title
+    property alias description: statusCommunityItem.subTitle
+    property bool joined: false
+    property int members: 0
+    property int notificationsCount: 0
+
+    property alias icon: statusCommunityItem.icon
+
+    property alias image: statusCommunityItem.image
+
+    implicitHeight: childrenRect.height + 12 * 2
+
+    StatusListItem {
+        id: statusCommunityItem
+        anchors.verticalCenter: parent.verticalCenter
+        anchors.left: parent.left
+        anchors.right: buttonsRow.left
+        anchors.rightMargin: 12
+        tertiaryTitle: qsTrId("-1-members").arg(members)
+    }
+
+    Row {
```

Comment:
You don't need that Row here. The StatusListItem can handle all of those with the `components` property. You just chuck the `StatusFlatButton`s in the array and they should appear in the Item at the end